pages/about.html

```diff
@@ -50,7 +50,7 @@ <h2 class="divider-style"><span><strong>About Us</strong></span></h2>
                     <p style="color: rgb(27,103,215);"><strong>AGGES: Who are we?</strong></p>
                 </div>
                 <div>
-                    <p style="padding-left: 5px;"><strong>Association of Geospatial Engineering Students</strong>&nbsp; is one of among the clubs at <strong>JKUAT.&nbsp; </strong>The&nbsp; club consist of like minded passionate Geomatic Engineering, Geospatial Engineering, and GIS
+                    <p style="padding-left: 5px;"><strong>Association of Geospatial Engineering Students</strong>&nbsp; is among the clubs at <strong>JKUAT.&nbsp; </strong>The&nbsp; club consist of like minded passionate Geomatic Engineering, Geospatial Engineering, and GIS
```

Comment:
- The AGGES Projects team will guide in wordings and chose of phrases

Reply:
Noted. will review this by the executive team and the dev team that we will form for maintenance